### DISCUSSION TITLE
Registering into mweb forum does not working

### DISCUSSION THREAD

**Discussion by lazna** [pawelsalawa/sqlitestudio#3412] [General]
### Details
Here you describe your problem details, or proposal for enhancement.

it give me HTTP error 500

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

Clicking to "Agree to thete terms
https://forum.sqlitestudio.pl/ucp.php?mode=register

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
Yes, forum is pending for "fixing".
